tdrs-backend/tdpservice/email/templates/stuck-file-list.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+{% block content %}
+
+
+
+<!-- Body copy -->
+<p style="color: #000000;">
+</p>
+
+<p>Hello,</p>
+
+<p>The system has detected stuck data submissions.</p>
+
+<table
+  style="width: 100%; background-color: #ffffff; border-width: 1px; border-color: #000000; border-style: solid; color: #000000; text-align: left; margin-bottom: 8px; font-size: 8pt;">
+  <thead style="border-style: solid; border-color: #000000; border-width: 1px; background-color: #e6e6e6;">
+    <tr>
+      <th style="padding: 4px; border-style: solid; border-color: #000000; border-width: 1px;">Stt</th>
+      <th style="padding: 4px; border-style: solid; border-color: #000000; border-width: 1px;">Section</th>
+      <th style="padding: 4px; border-style: solid; border-color: #000000; border-width: 1px;">Fiscal year</th>
+      <th style="padding: 4px; border-style: solid; border-color: #000000; border-width: 1px;">Submitted on</th>
+      <th style="padding: 4px; border-style: solid; border-color: #000000; border-width: 1px;">File</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for file in files %}
```

Comment:
What happens if we have hundreds of files stuck?

Reply:
Do we have limits on the content size of email we can send with sendgrid?

Reply:
these docs are for attachments, but mentions having a total email size limit of 30MB
https://www.twilio.com/docs/sendgrid/ui/sending-email/attachments-with-digioh#limitations